ibis/backends/postgres/registry.py

```diff
@@ -585,7 +585,8 @@ def _array_map(t, op):
             sa.func.unnest(t.translate(op.arg))
             # name the columns of the result the same as the lambda parameter
             # so that we can reference them as such in the outer query
-            .table_valued(op.param).render_derived()
+            .table_valued(op.param)
+            .render_derived()
```

Comment:
This may be an improvement over Black, as it's more aligned with what the change would be without the interleaved comment.